dev-java/rrd4j/rrd4j-3.8.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+MAVEN_ID="org.rrd4j:rrd4j:3.8.2"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="A high performance data logging and graphing system for time series data."
+HOMEPAGE="https://github.com/rrd4j/rrd4j/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/refs/tags/${PV}.tar.gz -> ${P}-sources.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="3.8"
+KEYWORDS=""
+
+DEPEND=">=virtual/jdk-1.8:*"
+RDEPEND=">=virtual/jre-1.8:*"
+
+S="${WORKDIR}/${P}"
+
+JAVA_SRC_DIR="src/main/java"
+JAVA_RESOURCE_DIRS=( "src/main/resources" )
+
+# needs org.ops4j.pax.exam
```

Comment:
But that's only one test in `src/test/java/org/rrd4j/osgi/OSGiSmokeTest.java` of 310 `@Test` in total.
There was similar in [dev-java/commons-compress](https://github.com/gentoo/gentoo/blob/master/dev-java/commons-compress/commons-compress-1.21-r1.ebuild#L89-L91) ...